xml/article_sap_trento.xml

```diff
@@ -0,0 +1,1052 @@
+            dedicated infrastructure resources. It provides a web front end.
+        </para>
+        <para>
+            The <emphasis role="strong">&t.agent;</emphasis> is a single background process (<systemitem>trento agent</systemitem>) running
+            in each host of the target infrastructure the user desires to monitor.
+            Except for third-party modules<!-- like Consul and ARA-->, all the components
+            are embedded within one single Trento binary.
+        </para>
+        <para>
+            See <xref linkend="fig-trento-architecture"/> for additional details.
+        </para>
+        <!-- toms 2021-11-26: Do we have a vector graphic for this?
+        https://github.com/trento-project/trento/blob/main/docs/trento-architecture.svg.
+        -->
+        <figure xml:id="fig-trento-architecture">
+            <title>Architectural overview</title>
+            <!--
+                replace SAP Application Platform Cluster with
+                SAP NetWeaver Cluster
+            -->
+            <mediaobject>
+                <imageobject>
+                    <imagedata width="80%" fileref="trento-architecture-outline.png"/>
+                    <!-- Source: https://confluence.suse.com/display/SAP/Architecture -->
+                </imageobject>
+            </mediaobject>
+        </figure>
+    </section>
+
+    <section>
+        <title>Product lifecycle and update strategy</title>
+        <!--
+            Most ist taken from https://confluence.suse.com/x/uoOyN
+        -->
+        <para>The product lifecycle is distinguished between the two main components:</para>
+
+        <variablelist>
+            <varlistentry>
+                <term>&t.agent;</term>
+                <listitem>
+                    <formalpara>
+                        <title>Delivery mechanism:</title>
+                        <para>RPM package for &productname; 15&nbsp;SP1 and newer.</para>
+                    </formalpara>
+                    <formalpara>
+                        <title>Supported runtime:</title>
+                        <para>
+                            Supports &productname; 15&nbsp;SP1 and newer.
+                        </para>
+                    </formalpara>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term>&t.server;</term>
+                <listitem>
+                    <formalpara>
+                        <title>Delivery mechanism:</title>
+                        <para>
+                            A set of container images from the &suse; registry together with a
+                            Helm chart for easy installation.
+                        </para>
+                    </formalpara>
+                    <formalpara>
+                        <title>&k8s;:</title>
+                        <para>
+                            The &t.server; runs on any current Cloud Native Computing Foundation (CNCF)
```

Comment:
Hmn, that might be the case. My intention was to have a high level overview in the "_Product lifecycle and update strategy_" and a more detailed part in the _Trento Server requirements_. Not sure if this will work in the end, but would prefer to leave it as is for the time being.